software/control/widgets.py

```diff
@@ -439,6 +481,25 @@ def init_ui(self):
         self.exposure_spinbox.valueChanged.connect(self.update_exposure_time)
         self.apply_button.clicked.connect(self.apply_settings)
```

Comment:
To be consistent with the general behavior of the GUI it's better to not have this button. Each value change should be updated immediately.